bucket/ratiomaster.json

```
@@ -0,0 +1,18 @@
+{
+    "version": "0.43",
+    "description": "Standalone application which fakes upload and download stats of a torrent",
+    "homepage": "https://ratiomaster.net/",
+    "license": "MIT",
+    "url": "https://github.com/NikolayIT/RatioMaster.NET/releases/download/0.43/RatioMaster.NET_0.43.zip",
+    "hash": "e3f415a2d9c7e9eccb926c85d78bb3acd3840990f25a946a894d00b0db7f6ba2",
+    "shortcuts": [
+        [
+            "RatioMaster.NET.exe",
+            "RatioMaster.NET"
+        ]
+    ],
+    "checkver": "github",
```

```suggestion
    "checkver": {
        "github": "https://github.com/NikolayIT/RatioMaster.NET"
    },
```
`"checkver": "github"` is only applicable if **"homepage"** is a Github repo